### PR TITLE
Added Capitalize in Haskell

### LINKED ISSUE #1356: Add Capitalize in Haskell
See requirements: https://sample-programs.therenegadecoder.com/projects/capitalize/
Add snippet here: https://github.com/TheRenegadeCoder/sample-programs/tree/master/archive/h/haskell

### PR DESCRIPTION
**Pull request by csrunner20**
Congrats on taking the first step to contributing to the Sample Programs repository maintained by [The Renegade Coder][1]! For simplicity, please make sure that your pull request includes one and only one contribution.

## Complete the Applicable Sections Below

Find which section best describes your pull request and make sure you fill it out. To start, let us know which issue you've fixed.

- [X] I fixed #1356 

### Code Snippets

- [X] I named the pull request using `Added/Updated <Sample Program> in <Language>` format
- [X] I created/updated the language README
  - [X] I added the sample program name to the README
  - [ ] I added fun facts (i.e. debut, developer, typing, etc.)
  - [ ] I added reference link(s) to the README
  - [ ] I added solution citations when necessary (see [plagiarism][2])

### Documentation

- [ ] I named the pull request using `Added/Updated <Sample Program> in <Language> Article` format
- [ ] I followed the applicable article template
  - [ ] I added/updated a [language article][4] (i.e. The Python Programming Language)
  - [ ] I added/updated a [code snippet article][3] (i.e. Hello World in Perl)
  - [ ] I added/updated a [project article][5] (i.e. Fizz Buzz in Every Language)

### Testing

- [ ] I named the pull request using `Added/Updated <Language>/<Project> Testing` format
- [ ] I followed the testinfo template, if applicable

## Notes

Feel free to describe what you added or updated.

[1]: https://therenegadecoder.com/
[2]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/.github/CONTRIBUTING.md#plagiarism
[3]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/CODE_ARTICLE_TEMPLATE.md
[4]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/LANGUAGE_ARTICLE_TEMPLATE.md
[5]: https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/templates/PROJECT_ARTICLE_TEMPLATE.md

### CODE REVIEW

File: archive/h/haskell/README.md [hunk 1]
```diff
@@ -7,8 +7,9 @@ Welcome to Sample Programs in Haskell!
 - [Baklava in Haskell](https://github.com/TheRenegadeCoder/sample-programs/issues/575)
 - [Bubble Sort in Haskell](https://github.com/TheRenegadeCoder/sample-programs/issues/800)
 - [Even Odd in Haskell](https://github.com/TheRenegadeCoder/sample-programs/issues/743)
+- [Capitalize in Haskell](https://github.com/TheRenegadeCoder/sample-programs/issues/1356)
 - [Factorial in Haskell](https://github.com/TheRenegadeCoder/sample-programs/issues/746)
-- [Fibonacci in Haskell](https://github.com/TheRenegadeCoder/sample-programs/issues/543)
+- [Fibonacci in Haskell](https://github.com/TheRenegadeCoder/sample-programs/issues/543)https://github.com/TheRenegadeCoder/sample-programs/issues/1356
```
Comment (jrg94) [inline archive/h/haskell/README.md]:
```suggestion
- [Fibonacci in Haskell](https://github.com/TheRenegadeCoder/sample-programs/issues/543)
```

File: archive/h/haskell/capitalize.hs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+module Main where 
+import Data.Char
+
+capitalize :: String -> String
+capitalize (head:tail) = toUpper head : map toLower tail
```
[the remark this answers is not in the document]
Comment (jrg94) [reply archive/h/haskell/capitalize.hs]: That's a good question! We currently don't have that behavior fully defined: https://sample-programs.therenegadecoder.com/projects/capitalize/

That said, I think we should go the route you mentioned. Only capitalize the first letter. Don't mess with the rest of the string.

[the remark this answers is not in the document]
Comment (auroq) [reply archive/h/haskell/capitalize.hs]: Agreed. @csrunner20 would you mind changing this so it leaves everything but the first letter unchanged?

File: archive/h/haskell/capitalize.hs [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+capitalize [] = []
+
+main = do  
+    putStrLn "Input string of choice to capitalize first letter"
+    inputStr <- getLine
```
[the remark this answers is not in the document]
Comment (csrunner20) [reply archive/h/haskell/capitalize.hs]: Oh, can do!  Didn't realize that is what you wanted.

File: archive/h/haskell/capitalize.hs (hunk 2 above)
[the remark this answers is not in the document]
Comment (csrunner20) [reply archive/h/haskell/capitalize.hs]: Yep can do!

File: archive/h/haskell/capitalize.hs (hunk 3 above)
[the remark this answers is not in the document]
Comment (auroq) [reply archive/h/haskell/capitalize.hs]: Ya, we are working on updating our documentation. We have recently standardized things more than previously.
Thank you!

**Comment by auroq** [comment]
Looks great. Thanks!

**Pull request merged**